### PR TITLE
asset_tracker_v2: bump linked release version of nRF Asset Tracker

### PR DESCRIPTION
**Pull request by coderbyheart**
nRF Asset Tracker has released support for nRF Connect SDK v2.1.x

See https://github.com/NordicSemiconductor/asset-tracker-cloud-docs/issues/533

### CODE REVIEW

**Comment by coderbyheart** [comment]
Double checked that new links work.

**Review by rlubos — APPROVED** [review]

File: doc/nrf/links.txt [hunk 1]
```diff
@@ -127,14 +127,14 @@
...
-.. _`nRF Asset Tracker web application`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v1.8.x/docs/app/Index.html
-.. _`Getting started guide for nRF Asset Tracker for AWS`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v1.8.x/docs/aws/GettingStarted/Index.html
-.. _`Getting started guide for nRF Asset Tracker for Azure`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v1.8.x/docs/azure/GettingStarted/Index.html
+.. _`Timestamping`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v2.1.x/docs/guides/FourKindsOfData.html#time-stamping
+
+.. _`nRF Asset Tracker project`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v2.1.x/docs/project/Index.html
+.. _`nRF Asset Tracker for AWS`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v2.1.x/docs/aws/Index.html
+.. _`nRF Asset Tracker for Azure`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v2.1.x/docs/azure/Index.html
+.. _`nRF Asset Tracker web application`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v2.1.x/docs/app/Index.html
```
Comment (peknis) [inline doc/nrf/links.txt:135]: And the same with these three. Not linked to from any NCS docs. Ok to remove.

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/links.txt (hunk 1 above)
Comment (coderbyheart) [inline doc/nrf/links.txt:135]:
```suggestion
```

File: doc/nrf/links.txt [hunk 2]
```diff
@@ -127,14 +127,11 @@
...
 
-.. _`Timestamping`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v1.8.x/docs/guides/FourKindsOfData.html#time-stamping
-
-.. _`nRF Asset Tracker project`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v1.8.x/docs/project/Index.html
-.. _`nRF Asset Tracker for AWS`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v1.8.x/docs/aws/Index.html
-.. _`nRF Asset Tracker for Azure`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v1.8.x/docs/azure/Index.html
-.. _`nRF Asset Tracker web application`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v1.8.x/docs/app/Index.html
-.. _`Getting started guide for nRF Asset Tracker for AWS`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v1.8.x/docs/aws/GettingStarted/Index.html
-.. _`Getting started guide for nRF Asset Tracker for Azure`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v1.8.x/docs/azure/GettingStarted/Index.html
+.. _`Timestamping`: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v2.1.x/docs/guides/FourKindsOfData.html#time-stamping
```
Comment (coderbyheart) [inline doc/nrf/links.txt:130]:
```suggestion
```

**Comment by rlubos** [comment]
@coderbyheart  Please squash the fixes.

**Pull request merged**